classad/classad.go

```diff
@@ -114,6 +114,15 @@ func ReadClassAds(r io.Reader) ([]ClassAd, error) {
 	return ads, nil
 }
 
+// MapStringStringToClassAd converts a map[string]string to a ClassAd.
```

Comment:
Can you include a note that this function will convert values to numeric types when appropriate?  Otherwise I could see someone expecting a ClassAd with everything set to `Type: String`.

Reply:
Done.  Please let me know if that looks good.

Reply:
Perfect, thanks!